index.html

```diff
@@ -305,6 +305,50 @@ <h5 class="about-me-heading">About Me</h5>
         </ul>
       </div>
     </section>
-    
+    <footer class="footer">
+      <h2 class="footer-text">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form action="https://formspree.io/f/xpzbopvv" method="POST">
+        <ul class="contact-section">
+          <li class="form-items">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              class="name"
+              maxlength="20"
+              placeholder="Full name"
+              required
+            />
+          </li>
+          <li class="form-items">
+            <input class="email" type="email" placeholder="Email address" required />
+          </li>
+          <li class="form-items">
+            <textarea
+              class="textarea"
+              name="textarea"
+              id="textarea"
+              cols="30"
+              rows="10"
+              maxlength="500"
+              placeholder="Enter your text here"
+              required
+            >
+            </textarea>
+          </li>
+        </ul>
+        <button type="submit" class="submit-button">Get in Touch</button>
+      </form>
+      <hr />
+      <div class="icons">
+        <ion-icon class="social-icon" name="logo-github"></ion-icon>
+        <ion-icon class="social-icon" name="logo-linkedin"></ion-icon>
+        <ion-icon class="social-icon" name="logo-twitter"></ion-icon>
+        <ion-icon class="social-icon" name="logo-whatsapp"></ion-icon>
+      </div>
```

Comment:
- [ ] Please increase the gap between these social media icons and make sure it matches the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/165702791-2523fe58-d326-47f1-b8a3-f446d4a8a33e.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/165702837-d6a7b930-d015-4753-9f04-786dcd1c6d37.png)